pkg/providers/launchtemplate/suite_test.go

```diff
@@ -976,28 +980,28 @@ var _ = Describe("LaunchTemplate Provider", func() {
 		})
 	})
 	Context("AL2", func() {
-		var info *ec2.InstanceTypeInfo
+		var info ec2types.InstanceTypeInfo
 		BeforeEach(func() {
 			var ok bool
-			var instanceInfo []*ec2.InstanceTypeInfo
-			err := awsEnv.EC2API.DescribeInstanceTypesPagesWithContext(ctx, &ec2.DescribeInstanceTypesInput{
-				Filters: []*ec2.Filter{
+			var instanceInfo []ec2types.InstanceTypeInfo
+			err := awsEnv.EC2API.DescribeInstanceTypesPages(ctx, &ec2.DescribeInstanceTypesInput{
```

Comment:
I don't think this is a real method on the real API so I think we can drop this in favor of just calling DescribeInstanceTypes on our fake